MAINTENANCE.md

```diff
@@ -6,7 +6,7 @@
 * Create pull request with the latest commit with bumped versions
 * Ask for merge of pull request. Once it is merged:
 * Checkout latest master tag it vX.Y.Z
-* Push the tag to master branch (`git push --tags`)
+* Push the tag to master branch (eg. `git push origin v.X.Y.Z`)
```

Comment:
- `origin` is misleading as it is dependent on remote setup. I personally have main repo marked as `upstream` and my fork as `origin`.
- `master branch` is also not correct

What about the following formulation:

Push the tag to the `gooddata/gooddata-python-sdk` repository (e.g. `git push <remote> v.X.Y.Z` where remote is bind to the `gooddata/gooddata-python-sdk` )

Reply:
You are right!